File: curso-python/ex075.py
valores = []
for i in range(0,4):
    n = int(input(f"digite o {i+1} valor: "))
    valores.append(n)

tupla = tuple(valores)

print(f"você digitou os valores: {tupla}")
print(f"O valor 9 apareceu {tupla.count(9)} vez(es)")

if 3 in tupla:
    print(f"O valor 3 apareceu na {tupla.index(3)+1}°a posição")
else:
    print("o valor 3 não foi digitado")

par = 0
for valor in tupla:
    if valor % 2 == 0:
        par += 1


print(f"os valores pares digitados foram {par}")

File: javascript-exercicios/ex075.js
const prompt = require("prompt-sync")()
let valores = []
for (let i=0; i < 4; i++){
    let n = parseInt(prompt(`digite o ${i+1}° valor: `))
    valores.push(n)
}

console.log(`você digitou os valores: [${valores}]`)
quant9 = valores.filter(valor => valor == 9).length

console.log(`o valor 9 apareceu ${quant9} vezes`)

const localizarPos3 = valores.indexOf(3)
if (localizarPos3 != -1){
    console.log(`o valor 3 apareceu na ${localizarPos3 + 1}° posição`)
}

let par = 0
for (let v of valores){
    if (v % 2 == 0){
        par += 1
    }
}

console.log(`o(s) valor(es) par(es) digitado(s) foi ${par} valor(es)`)